src/player/Player.php

```diff
@@ -1530,6 +1531,13 @@ public function onUpdate(int $currentTick) : bool{
 				$this->fireTicks = 1;
 			}
 
+			$chargeable = $this->getInventory()->getItemInHand();
+			if($chargeable instanceof Chargeable && $this->isUsingItem()){
+				if($chargeable->continueCharge($this, $this->getItemUseDuration())){
```

Comment:
I feel like `continueCharge` is weirdly specific. We probably need this for general `Releasable` anyway if we want to make eating sounds server-sided. So it seems sus to make a special case for this